### PR TITLE
another dono mask

### PR DESCRIPTION
**Pull request by CandleJaxx**
we do a little bit of gaming
image

(as far as im aware the dono hasn't gone through yet, but i'm uploading it earlier incase it needs edits)

### CODE REVIEW

**Comment by CandleJaxx** [comment]
keep yourself safe

File: modular_nova/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 1]
```diff
@@ -918,6 +918,77 @@
...
+		alternate_worn_layer = BACK_LAYER
+
+/obj/item/clothing/mask/gas/psycho_malice/alt_click_secondary(mob/user)
+	. = ..()
+	if(.)
+		return
+	if(user.can_perform_action(src, NEED_DEXTERITY))
+		adjust_mask(user)
+
+/obj/item/clothing/mask/gas/psycho_malice/proc/adjust_mask(mob/living/carbon/human/user)
```
Comment (GoldenAlpharex) [inline modular_nova/modules/customization/modules/clothing/~donator/donator_clothing.dm:972]: Document this proc please.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Other than that, this will remain on hold until the donation happens.

**Review by GoldenAlpharex — APPROVED** [review]
Let's go!

**Pull request merged**